Dispatcher/Controllers/ArticleModel.swift

```diff
@@ -1,26 +1,38 @@
 import Foundation
 
-protocol Article {
-    var title: String { get }
-    var date: Date { get }
-    var url: String { get }
-    var isFavorite: Bool { get set }
-}
-
-struct ArticleModel: Article {
-    var title: String
-    var date: Date
-    var url: String
-    var isFavorite: Bool
+struct ArticleModel: Codable {
```

Comment:
You have struct named ArticleModel and Article.
I cant understand the purpose from the name. Lets talk tomorrow.

Reply:
Explained in conversation, got separated into "ArticleModel" (in Models dir) and "ArticleResponse" (in Networking > Responses dir) inside the MVVM pull request
@abenartz